src/openrct2/drawing/Text.cpp

```diff
@@ -141,12 +141,12 @@ static void DrawTextEllipsisedCompat(
     DrawText(dpi, x, y, &_legacyPaint, buffer);
 }
 
-void gfx_draw_string(rct_drawpixelinfo* dpi, const_utf8string buffer, uint8_t colour, int32_t x, int32_t y)
+void gfx_draw_string(rct_drawpixelinfo* dpi, const_utf8string buffer, uint8_t colour, int32_t x, int32_t y, TextFont font)
```

Comment:
Since the vast majority of callers use Font medium maybe that should be a default value.

Reply:
I also wonder if I should perhaps rename it to TextSettings/TextOptions and put the color in there as well

Reply:
I agree with Duncan. It makes for more readable code IMO.